subsys/bluetooth/controller/hci/hci.c

```diff
@@ -1686,8 +1703,48 @@ static void le_advertising_report(struct pdu_data *pdu_data, u8_t *b,
 		data_len = 0;
 	}
 
+	rssi = b[offsetof(struct radio_pdu_node_rx, pdu_data) +
+		 offsetof(struct pdu_adv, payload) + adv->len];
+
+#if defined(CONFIG_BT_CONTROLLER_PRIVACY)
+	if (direct) {
```

Comment:
Btw, this is one easy way to combat excessively long lines of code since it eliminates at least one indentation level.